include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -835,13 +863,21 @@ def AIE_DMABDOp: AIE_Op<"dma_bd", []> {
   let hasVerifier = 1;
 
   let assemblyFormat = [{
-    `(` $buffer `:` type($buffer) (`,` $offset^)? (`,` $len^)? (`,` $dimensions^)? `)` attr-dict
```

Comment:
I think moving offset and len to the attribute dictionary should be part of another PR, centred around syntax.

Reply:
Sure